nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-json-record-utils/src/main/java/org/apache/nifi/json/WriteJsonResult.java

```diff
@@ -19,6 +19,7 @@
 
```

Comment:
I think in the section of WriteJsonResult from line 176 to 189, the scientific notation logic needs to be added for cases where the Record is already serialized.

Reply:
Got it. Thanks @Lehel44 that makes sense. Great catch! Will update shortly, with new test cases.